webapp/channels/src/components/channel_select/channel_select.tsx

```diff
@@ -28,6 +29,8 @@ const ChannelSelect = ({
 }: Props) => {
     const intl = useIntl();
 
+    const activeChannels = channels.filter((channel: Channel) => channel.delete_at === 0);
```

Comment:
@TheInvincibleRalph This should be done in the `getMyChannelsSorted` selector in the index.ts file of this directory.

Reply:
Alright @BenCookie95. Thank you for the review.